.gitmodules

```diff
@@ -15,7 +15,7 @@
 	url = https://github.com/SuperTux/sexp-cpp.git
 [submodule "data"]
 	path = data
-	url = https://github.com/SuperTux/data.git
+	url = https://github.com/wansti/data.git
```

Comment:
Probably not. Sorry I thought I removed that. 

On 14 August 2016 7:31:22 PM AEST, "M. Teufel" <notifications@github.com> wrote:
>> @@ -15,7 +15,7 @@
>>  	url = https://github.com/SuperTux/sexp-cpp.git
>>  [submodule "data"]
>>  	path = data
>> -	url = https://github.com/SuperTux/data.git
>> +	url = https://github.com/wansti/data.git
>
>We probably don't want this in master.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/SuperTux/supertux/pull/548/files/c886fcc79a1ae4b90592432f9a1508189c64e434#r74698720

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.